packages/cli/src/daemon-config.ts

```diff
@@ -90,7 +89,7 @@ export class DaemonHTTPApiConfig {
    */
   @jsonMember(AnyT, {serializer: inPort => {
       const validPort = Number(inPort)
-      if (inPort == undefined || inPort == null) {
+      if (inPort == null) {
```

Comment:
@willex how is it relevant to stream controllers?

Reply:
It is not related. It was a left over suggestion from a previous PR that I overlooked and put it in here as a chore vs. opening a new PR for a small change.